src/v1/controllers/VerifyWebauthnController.js

```diff
@@ -101,13 +101,15 @@ export default FormController.extend({
 
           // AbortController is not supported in IE11
           // eslint-disable-next-line compat/compat
```

Comment:
Do we still want the linting disabled here?

Reply:
removed